keyboards/gh60/satan/config.h

```diff
@@ -20,11 +20,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x6060
+#define VENDOR_ID       0x4335 // GEEKhack
```

Comment:
The Satan isn't made by Geekhack.

Reply:
@fauxpark What do you think about removing the inline comment, but leaving the VID as-is?

Reply:
It's pretty much been lumped as a "version" of the GH60, and it has since been [sold by 3rd-party sellers on Taobao](https://geekhack.org/index.php?topic=41464.msg1620728#msg1620728) (and now with many other evndors), so it could be said that there is no specific "vendor". The GH60 Satan is _based_ on the original footprint of the GH60, but that's all we know about the "vendor" for these.

Reply:
[Removed inline comment ](https://github.com/qmk/qmk_firmware/pull/8442/commits/55cb8a58097d221751c6ce02a053d8a91ecfa5e9)

Reply:
As I understand it, Satan is the "vendor" in this case - my PCB has the URL satan.com.cn on the front (though, it's a dead link).

Reply:
@fauxpark Is it really worth setting a new VID at this point when it's a defunct "vendor"? There certainly won't be more GH60s besides what's currently in place, I would think.